serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc

```diff
@@ -23,6 +23,11 @@ A window appears containing your current operating system as selected and a link
 --
 You might need to grant permissions to run the file depending on your operating system and settings.
 
+[NOTE]
+====
+If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 
```

Comment:
```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the MacOS' approach towards security and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 
```

Reply:
@spolti , per Apple[1] [2], it is macOS since 2016..

[1] - https://www.apple.com/sg/macos/ventura/
[2] - https://en.wikipedia.org/wiki/MacOS

Reply:
@bbalakriz didn't understand your comment :)
I am not saying it isn't on mac since.... I just suggested a very small rephrase :)